docs/advanced/verifying-binaries.md

```diff
@@ -1,53 +1,305 @@
+it from a familiar website.
+
+Although binaries make it easy to install software, binary code is not
+human-readable. This carries an inherent risk, because it’s nearly impossible
+to tell if the binary was compiled from the true source code, or if the source
+code was tampered with by an adversary first.
+
+Consider the case where an adversary created malware and disguised it to look
+like a Decred binary. Adversaries can try to trick you into running their
+malware by setting up fake download locations, phishing, or a MITM attack. In
+theory, even if you download the binary from somewhere you trust (like Github),
+an attacker could still potentially intercept and replace your download with
+their own malware. If you run the binary without verifying that it isn’t
+malware, you expose yourself to security risks.
+
+### How do you verify that a binary hasn’t been tampered with?
+
+The core concept of [file verification](https://en.wikipedia.org/wiki/File_verification)
+relies on hashing. The Decred team takes it a step further and uses
+[GnuPG](https://gnupg.org/index.html) to sign their hashes so you can verify
+that the file came directly from the Decred team.
+
+When the Decred team [compiles source code into binaries](https://github.com/decred/decred-release),
+they hash each binary using [SHA-256](https://en.wikipedia.org/wiki/SHA-2).
+Each hash is stored in a `manifest.txt` file. Then they sign the manifest using
+their public GnuPG key. This creates a signature file, ending in
+`manifest.txt.asc`.
+
+To check if your downloaded binary is a 100% match to the official Decred
+binary, you need to do two things:
+
+1. **Verify that the manifest signature was directly signed by Decred:**
+   Download the `manifest.txt` and the signature `manifest.txt.asc`. Use GnuPG
+   to check the signature against the public Decred Release Signing Key.
+
+     * SUCCESS: If the signatures match, you know you can trust that the
+       manifest came directly from Decred, and that the hashes inside it are
+       trustworthy.
+
+     * FAIL: If the signatures don't match, then your manifest is not the
+       official Decred manifest. Do not continue, delete the files you've
+       downloaded, and check for security problems before trying again.
+
+2. **Verify that the binary hash matches the manifest hash:** Hash your
+   downloaded binary and verify that its hash matches the hash in the official
+   Decred `manifest.txt`.
+
+     * SUCCESS: If your hash matches the hash in the manifest, you know you can
+       trust and run the binary.
+
+     * FAIL: If your hash doesn't match, your binary doesn't match the official
+       Decred binary release, and you should not continue. Delete the binary
+       you've downloaded, and check for security problems before trying again.
+
+## Detailed Instructions
+
+### Preliminary Steps
+
+First, go to the official [Decred releases](https://github.com/decred/decred-release/releases)
```

Comment:
Sounds good.

Just FYI, https://decred.org/wallets/ links the Command-line app suite to the `dcrinstall` repo.
I think there should be an separate issue that sorts out all download links and perhaps removes all remnants of links to `dcrinstall`? I don't know if I'm the right person to write that issue, though, because I don't know the history and the nuance.

Reply:
Yeah, thats why I said "**kind of**". Its also the recommended installer in docs.decred.org - https://docs.decred.org/wallets/cli/cli-installation/

Its a wider discussion than just updating links.